### PR TITLE
Add release 5.0 post

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/news/release-5.0.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+- Use system gamemode client if possible by @Scrumplex in PolyMC/PolyMC#1210
+
+### Removed
+
+- Remove outdated information about "upcoming" mod updater by @DioEgizio in PolyMC/PolyMC#1185
+- Remove unused libraries by @DioEgizio in PolyMC/PolyMC#1208
+
+Full Changelog: <https://github.com/PrismLauncher/PrismLauncher/compare/1.4.2...5.0>
+
+You can [grab the latest download here](https://github.com/PolyMC/PolyMC/releases/tag/5.0) for your respective platform.
```
Comment (forkiesassds) [inline src/news/release-5.0.md:92]: PolyMC URL, should be changed to Prism Launcher URL.

Comment (Scrumplex) [reply src/news/release-5.0.md:92]: oh right, thanks

**Comment by Scrumplex** [comment]
> Change the minimum MacOS requirement to 10.13 and add that there is now a MacOS Legacy Package.

The macOS requirement is for the auto updater, which doesn't exist yet for the legacy package

**Review by cozyGalvinism — APPROVED** [review]
Looks good

**Review by CodexNotFound — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Pull request merged**